### PR TITLE
Make BTable sortable column header text non-selectable

### LINKED ISSUE #2301: BTable sortable table header has selectable text
### Overview of the problem

**Buefy** version: [0.8.12]
**Vuejs** version: [2.6.16]
**OS/Browser**: Chrome

### Description

Double-clicking on a sortable table header selects the header text. From a standard UX perspective, text in control-based elements (buttons etc.) that can be clicked multiple times should have non-selectable text. A sortable table header is a control-based element (which is also double-clicked quite often to change sorting direction).

Demo:
buefy_table_selection_none_demo

### Steps to reproduce

Double-click a sortable table header.

### Expected behavior

Text should not have been selected.

### Actual behavior

Text was selected.

### PR DESCRIPTION
**Pull request by raimondlume**
Fixes #2301 

## Proposed Changes
Add `user-select: none;` to sortable table header to make it non-selectable.

### CODE REVIEW

**Comment by service-paradis** [comment]
You can also use this [mixin](https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L133) for mobile and cross-browser support.

**Comment by jtommy** [comment]
@raimondlume Ok but add a prop on table column component because it's a breaking change

File: src/components/table/TableColumn.vue [hunk 1]
```diff
@@ -27,6 +27,10 @@ export default {
         subheading: [String, Number],
         customSort: Function,
         sticky: Boolean,
+        headerSelectable: {
+            type: Boolean,
+            default: true
```
Comment (jtommy) [inline src/components/table/TableColumn.vue]: No It's a breaking, so you have to set to false at the moment

Comment (service-paradis) [reply src/components/table/TableColumn.vue]: Seems OK, since the prop determine if the header is selectable. `true` is the current behavior.

Comment (raimondlume) [reply src/components/table/TableColumn.vue]: The style binding is inverted (`'is-unselectable': !column.headerSelectable`), so if default is True, no style is added. This should not be breaking?

**Comment by jtommy** [comment]
You are right, I'm sorry

**Comment by raimondlume** [comment]
@jtommy Should this prop default to `false` with the next breaking update? If yes, should I make a PR for it?

**Comment by jtommy** [comment]
Yes

**Comment by raimondlume** [comment]
> Yes

Added the breaking change: #2306

**Pull request merged**